CHANGES.txt

```diff
@@ -35,6 +35,8 @@ Trunk
     FALCON-1770 Update README file (Ajay Yadava)
 
   BUG FIXES
+    FALCON-1842 Falcon build failed in Jenkins at OozieFeedWorkflowBuilderTest (Balu Vellanki)
```

Comment:
CHANGES.txt doesn't need to be added. As we can have a script to generate the change log, PR 34